internal/datastore/common/revisions/optimized_test.go

```diff
@@ -31,6 +32,7 @@ var (
 )
 
 func TestOptimizedRevisionCache(t *testing.T) {
+	zerolog.SetGlobalLevel(zerolog.Disabled)
```

Comment:
This has always bothered me about zerolog. Is there no way to control the zerolog level without calling a global singleton? It leads to things like: "if you run these tests in this order you get no logs, if you run only one of the tests you get logs"

Reply:
Each logger instance can have its own logging level, for sure. I can give it another spin in a different PR so it does not block this one